manager/manager.go

```diff
@@ -509,6 +533,27 @@ func (m *manager) destroyContainer(containerName string) error {
 		})
 	}
 	glog.Infof("Destroyed container: %q (aliases: %v, namespace: %q)", containerName, cont.info.Aliases, cont.info.Namespace)
+
+	contRef, err := cont.handler.ContainerReference()
+	if err != nil {
+		return err
+	}
+
+	contSpecs, err := cont.handler.GetSpec()
```

Comment:
This will fail since the container doesn't exist anymore. Lets leave it blank for now. We can consider caching the last spec we got and returning that.